scripts/us_eia/opendata/process/common.py

```diff
@@ -21,7 +21,10 @@
 from sys import path
 
 # For import util.alpha2_to_dcid
-path.insert(1, '../../../../')
+# path.insert(1, '../../../../')
+path.insert(1, '/usr/local/google/home/chandaluri/data/')
+path.insert(2,
+            '/usr/local/google/home/chandaluri/data/scripts/us_eia/opendata/')
```

Comment:
You should not have a hard coded local path, please make general. Please modify the code as suggested.